radicle-httpd/src/api/json.rs

```diff
@@ -0,0 +1,68 @@
+//! Utilities for building JSON responses of our API.
```

Comment:
we don't use comments like this, might wanna remove it.

Reply:
We definitely welcome module documentation though :)

Reply:
Yup, another case of keeping everything uniform.